biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -216,11 +220,13 @@ module FatesPlantHydraulicsMod
   real(r8), parameter :: th_sat_vg = 0.65_r8
   real(r8), parameter :: th_res_vg = 0.15_r8
   real(r8), parameter :: psd_vg    = 2.7_r8
+  real(r8), parameter :: m_vg      = 0.62963_r8
   real(r8), parameter :: tort_vg   = 0.5_r8
+  real(r8), parameter :: plant_tort = 0.0_r8
 
   ! The maximum allowable water balance error over a plant-soil continuum
   ! for a given step [kgs] (0.1 mg)
-  real(r8), parameter :: max_wb_step_err = 1.e-7_r8 
+  real(r8), parameter :: max_wb_step_err = 2.e-6_r8 
```

Comment:
I feel the tolerances that we've specified up to this point have been fairly arbitrary, and some of them were high enough to obscure errors. 1 e-7 is a tenth of a milligram per plant, now it is 2 milligrams per plant.  I honestly don't know what the right answer is here.  Plants can be small and the total water flux in a 30 minute time-step could be around a milligram, no? The intent of the tolerances are to ensure that the numerical integration's truncation error (solution error) is small within reason.  When its large it could potentially allow errors from other sources to pass by undetected.  
@JunyanDing , is this increased tolerance still required for runs to complete?  If it is, I suppose we should move forward for now, but create an issue to further evaluate why the errors are large.